src/services/services.ts

```diff
@@ -650,7 +650,7 @@ namespace ts {
         return firstDefined(getAllSuperTypeNodes(classOrInterfaceDeclaration), superTypeNode => {
             const baseType = checker.getTypeAtLocation(superTypeNode);
             const symbol = isStaticMember
-                ? find(checker.getExportsOfModule(baseType.symbol), s => s.escapedName === declaration.symbol.name)
+                ? baseType.symbol && find(checker.getExportsOfModule(baseType.symbol), s => s.escapedName === declaration.symbol.name)
                 : checker.getPropertyOfType(baseType, declaration.symbol.name);

```

Comment:
From discussing this with @DanielRosenwasser, this should actually be:

```ts
            let baseType = checker.getTypeAtLocation(superTypeNode);
            if (isStaticMember && baseType.symbol) {
                baseType = checker.getTypeOfSymbol(baseType.symbol)
            }
            const symbol = checker.getPropertyOfType(baseType, declaration.symbol.name);
```

`getTypeAtLocation` returns the instance type of a class symbol because that's what most people want. In the case of a static member, we need to call `getTypeOfSymbol` to get the class type of class symbol instead. After that, resolving the property by name works the same.